the-telephone-app-fe/src/components/Draw/Draw.js

```diff
@@ -292,30 +292,38 @@ const Draw = ({ width = "815rem", height = "350rem" }) => {
         <div className="col-2">
           <div className="row mt-5">
             <div className="col-12 mt-5">
-              <div className="card app-bg mw-100 border-4 mt-5">
+              <div className="card custom-border-color app-bg mw-100 border-4 mt-5">
                 <div className="card-body">
                   <div className="row">
-                    {drawModes.map((mode, index) => (
-                      <div key={index} className="col-6">
-                        <div
-                          style={
-                            mode.id === drawModeId
-                              ? { border: "solid 2px" }
-                              : {}
-                          }
-                          className="draw-mode-square ms-3"
-                        >
-                          <div className="draw-mode-inside-square">
-                            <div
-                              onClick={() => changeMode(mode)}
-                              className="ps-2 pt-2"
-                            >
-                              {mode.modeIcon}
+                    {drawModes.map((mode, index) =>
+                      mode.id === drawModeId ? (
```

Comment:
check this to add class inside className intead of here to avoid repeating the whole div just to change the class.
https://dev.to/cesareferrari/conditionally-assign-a-css-class-to-a-react-component-jk5